api/envoy/api/v2/auth/cert.proto

```diff
@@ -433,13 +433,16 @@ message DownstreamTlsContext {
 
     // Config for fetching TLS session ticket keys via SDS API.
     SdsSecretConfig session_ticket_keys_sds_secret_config = 5;
+
+    // Config to disable TLS session tickets altogether
```

Comment:
nit: please end with a period. Also, for my understanding, what is the behavior previously if none of the previous session ticket options were set? Should this be clarified in the docs a bit more? cc @ggreenway

Reply:
Agreed that previous behavior should be documented. I'm 95% sure that the behavior is that envoy/boringssl generates a key internally and can reuse tickets generated by this envoy, but cannot across a hot-restart, or if there is a pool of envoys behind an L3/L4 load balancer (unless it goes back to the originating envoy).

Reply:
I think that behaviour is documented here already https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/cert.proto#auth-tlssessionticketkeys
Do you think we should document here as well?

Reply:
@rulex123 Please copy (and slightly rework) the chunk you linked to, and comment that it's the behavior when this new setting is false.

Reply:
@ggreenway I expanded on the previous explanation, hope it makes the API clearer.